mlflow/java/scoring/src/main/java/org/mlflow/sagemaker/MLeapPredictor.java

```diff
@@ -47,7 +48,18 @@ public MLeapPredictor(String modelDataPath, String inputSchemaPath) {
 
     this.pipelineTransformer = bundleBuilder.load(new File(modelDataPath), mleapContext).root();
     try {
-      this.inputSchema = schemaReader.fromFile(inputSchemaPath);
+      StructType trainingSchema = schemaReader.fromFile(inputSchemaPath);
```

Comment:
To echo @kevinykuo's point in #1278, is it possible to modify `PandasSplitOrientedDataFrame.toLeapFrame` (https://github.com/mlflow/mlflow/blob/9ee43f8a180c72517b0c9ee82042d6eb90cd8d85/mlflow/java/scoring/src/main/java/org/mlflow/sagemaker/PandasSplitOrientedDataFrame.java#L91) so that `inputSchema` is not required?

This seems doable, as `inputSchema` is currently only used to select entries from a `<String, Object>` hashmap. What are your thoughts?

Reply:
I added a comment on https://github.com/mlflow/mlflow/issues/1278#issuecomment-496175204, let me know what you think.

Reply:
@ancasarb I responded to your comment on #1278 with a couple of clarifying questions.